permissions-fixer/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "nexus-fixer"
+version = "1.0.0"
+authors = ["Henry Genet <henry.genet@epfl.ch>"]
+edition = "2018"
```

Comment:
2018?

Reply:
It's the Toml manifest format, 2018 is the latest version